strings/strings/gl.yml

```diff
@@ -0,0 +1,594 @@
+name: Galician
+natively: Galiciano
+
+
+authors:
+ - D3KRISH
+
+# Commons
+com_1: "Inicie o proceso ..."
+com_2: "A busca comeza ..."
+com_3: "entrada non válida"
+com_4: "Medios non capaces .."
+com_5: "Descargar ..."
+com_6: "Cargando ..."
+com_7: "algo desordenado"
+
+#Chatactions
+can_1: "#GBanned_user unido. \ N \ n usuario: {} \ n razón: {} \ n \ n o usuario estaba prohibido"
+can_2: "∆ #USERNAMEUPDATE \ n \ n @{} cambiou o nome de usuario en @{}"
+can_3: "∆ #USERNAMEUPDATE \ n \ n {} eliminou o nome de usuario (@{})"
+can_4: "∆ #USERNAMEUPDATE \ n \ n {{} novo nome de usuario -> @{}"
+    
+# helpa
+help_1: "{} non é un complemento válido!"
+help_2: "Bot non respondeu á consulta en liña. \ N {} proba o comando de reinicio!"
+help_3: "Por favor, acenda o modo en liña para o seu bot @Botfather."
+help_4: "• complementos"
+help_5: "• Adons"
+help_6: "•• Conversa sonora"
+help_7: "complementos en liña ••"
+help_8: ": Gear: propietario Equipment"
+help_9: "Configuración: engrenaxe:"
+help_10: "•• Cʟᴏꜱᴇ ••"
+help_11: "Nome do complemento -{} \ n \ n✘ comandos dispoñibles -\ n \ n"
+help_12: "A chamada de voz non comeza"
+help_13: "Vostede fai '{} setDb Addons True" para que os complementos poidan vir "
+
+# userlogs
+userlogs_1: "O ID de chat que introduciu é incorrecto, corríxelo"
+userlogs_2: "Póñame a etiqueta a xente en grupo para que a xente poida ser"
+userlogs_3: "Chat" "
+userlogs_4: "O teu barco non pode mensaxes no grupo de rexistro de etiquetas. \ N Engádeo ao rexistro de etiquetas .."
+userlogs_5: "Esquerda {}"
+
+# admintools
+adm_1: "Non sei que o sexa"
+adm_2: "Hmm .. Creo que non teño dereitos aquí"
+adm_3: "Dime o tempo ..."
+pro_1: "promove respondendo a unha persoa"
+pro_2: "{} é agora o administrador {} co nome {}"
+de_1: "DeMote respondendo a unha persoa"
+de_2: "{} xa non no administrador {}"
+ban_1: "Ban o ID do usuario ou o nome de usuario"
+ban_2: "Non podo prohibir ao meu fabricante"
+ban_3: "Non teño dereito a prohibir ao usuario."
+ban_4: "{} {} prohibido {} en {}"
+ban_5: "\ n Reason: {}"
```

Comment:
Seems like you used some kind of translator.
Fix \ n and \ N present in file..

Reply:
Yes